task.sql

```diff
@@ -4,30 +4,66 @@ CREATE DATABASE ShopDB;
 USE ShopDB;
 
 CREATE TABLE Countries (
-    ID INT,
+    ID INT AUTO_INCREMENT,
     Name VARCHAR(50),
     PRIMARY KEY (ID)
 );
 
+CREATE TABLE Cities (
+		ID INT AUTO_INCREMENT,
+		Name VARCHAR(50),
+		CountryID INT,
+		FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
```

Comment:
The foreign key constraint `ON DELETE NO ACTION` might cause issues if a country is deleted while cities still reference it. Consider using `ON DELETE CASCADE` if you want to automatically delete dependent cities when a country is deleted.